lib/model/frames/form.js

```diff
@@ -140,11 +140,18 @@ Form.Partial = class extends Form {};
 
 Form.Extended = class extends Frame.define(
   'submissions',        readable,       'lastSubmission', readable,
+  'received',           readable,       'hasIssues',       readable,
+  'edited',             readable,
   'excelContentType',   readable
 ) {
   forApi() {
     return {
       submissions: this.submissions || 0,
+      reviewState: {
+        received: this.received || 0,
```

Comment:
Everywhere else in the API, the received review state is represented as `null` rather than as `'received'`. I'd love to somehow use `null` here as well, but I can't think of a good way to do so. Probably no change needed, just wanted to throw out this thought.